torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -33,9 +33,11 @@ using namespace torch::distributed::autograd;
 
 namespace {
 
-py::object toPyObj(IValue value) {
+SerializedPyObj toSerializedPyObj(IValue value) {
```

Comment:
It's only used by `MessageType::PYTHON_RREF_FETCH_CALL`.

Can we move under case `MessageType::PYTHON_RREF_FETCH_CALL` and make it a lambda, just like the markComplete lambda on line 116.

Reply:
Sure, edited, but the downside is that we will need to capture this lambda in two nested callbacks. I am not sure if the extra overhead worth it here.

Reply:
@mrshenli 

> If there is no capture by value, the closure type is typically trivially copyable and trivially moveable.
http://www.cplusplus.com/forum/general/202662/